mrmustard/lab/representations/data/gaussian_data.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import numpy as np
+
+from typing import Optional, TYPE_CHECKING, Union
+
+from mrmustard.lab.representations.data.matvec_data import MatVecData
+from mrmustard.math import Math
+from mrmustard.typing import Matrix, Scalar, Tensor, Vector
+
+
+if TYPE_CHECKING:  # This is to avoid the circular import issu with GaussianData<>QPolyData
```

Comment:
This type checking is not needed anymore.

Reply:
done